ansible_mcp_tools/server.py

```diff
@@ -30,12 +39,44 @@ def init_app_authentication_backend(self, app: Starlette):
                 LightspeedAuthenticationMiddleware, backend=self._auth_backend
             )
 
+    @override
     def sse_app(self, mount_path: str | None = None) -> Starlette:
         app = super().sse_app(mount_path=mount_path)
         self.init_app_authentication_backend(app)
         return app
 
+    @override
     def streamable_http_app(self) -> Starlette:
         app = super().streamable_http_app()
         self.init_app_authentication_backend(app)
         return app
+
+
+class LightspeedOpenAPIAAPServer(LightspeedBaseAAPServer):
+    def __init__(
+        self,
+        name: str,
+        service_name: str,
+        auth_backend: AuthenticationBackend | None,
+        spec_loader: SpecLoader,
+        **settings: Any,
+    ):
+        super().__init__(name, auth_backend, **settings)
+        tool_name_strategy = DefaultToolNameStrategy()
+        spec = spec_loader.load()
+        tool_parser = DefaultToolParser(spec, service_name, tool_name_strategy)
+        self.tools = tool_parser.parse_tools()
```

Comment:
maybe make it a private property
```suggestion
        self._tools = tool_parser.parse_tools()
```